types/jsonstream-next/jsonstream-next-tests.ts

```diff
@@ -0,0 +1,14 @@
+
+import json = require('jsonstream');

```

Comment:
Shouldn't that be testing jsonstream-next instead?

Reply:
```suggestion
import json = require('jsonstream-next');
```